concordium-consensus/src/Concordium/MultiVersion.hs

```diff
@@ -963,6 +971,9 @@ receiveTransaction transactionBS = do
 importBlocks :: FilePath -> MVR gsconf finconf UpdateResult
 importBlocks importFile = do
     vvec <- liftIO . readIORef =<< asks mvVersions
+    -- reset the stop flag to False, in case it was set to True
```

Comment:
Why is this the correct behaviour?

This flag is only set upon Ctrl+C/SIGINT. So how could they accidentally set this flag?

Reply:
This is a more proper approach. Consider a case where there is a need to start import process more than once. That would not be possible without resetting the flag.